packages/migrations/src/migrateDockerNetworkIpRange/restoreContainersToNetwork/recreateContainersToRecreate.ts

```diff
@@ -0,0 +1,32 @@
+import { docker, dockerComposeUp } from "@dappnode/dockerapi";
+import { logs } from "@dappnode/logger";
+import { filterContainers } from "./filterContainers.js";
+
+export async function recreateContainersToRecreate(
+  containersToRecreate: string[]
+): Promise<void> {
+  if (containersToRecreate.length > 0) {
+    logs.info(
+      `Recreating docker containers that require to be recreated: ${containersToRecreate}`
+    );
+    const composeFilesPathsToRecreate = (
+      await Promise.all(
+        filterContainers(containersToRecreate).map(async (cn) => {
+          // get the compose file path
+          return (await docker.getContainer(cn).inspect()).Config.Labels[
```

Comment:
This could be another function called `getComposeFileFromContainer`? Also, we must be sure that the file path of the docker compose is the one inside the dappmanager (and not the one in the host). I am pretty sure this code is ok, but just in case